### DISCUSSION TITLE
Mobile app triggers abuse detection while browsing

### DISCUSSION THREAD

**Discussion by chkn** [community/community#3804] [Mobile]
While browsing through repos on the mobile app- completely normal behavior- I found that sometimes tapping the back button shows the "Something went wrong" screen. I continued to go back to the Home Screen and I got this...

![image](https://user-images.githubusercontent.com/289291/115129341-fd717b80-9fb2-11eb-92b7-15be281e3df5.png)

Furthermore the button to share bug reports in the app is not working. I found a link here from the "App Support" link on the App Store.

**Comment by eliperkins** [comment]
Thanks for reporting this @chkn!

I think we had some wires crossed on backend when you ran into this, which included causing the bug report mechanism not work. It should be addressed at this point, but please reach back out if you continue to run into this and we can dig in deeper here!

**Comment by duchenean** [comment]
Hello,

Still getting this error to this day :

![IMG_2432](https://user-images.githubusercontent.com/9050997/136460333-a9f26fea-21ed-41f7-a712-30fc266d21a0.PNG)

Exactly as OP described, tapping back button trigger a 403 error.

**Reply by duchenean** [reply]
Also bug reporting in iOS app is broken

**Reply by chkn** [reply]
Yep, I just got that the other day too...
![IMG_2101](https://user-images.githubusercontent.com/289291/137987272-d4b472e4-0d61-4a5e-9f79-38f2708cc845.png)
